<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientSummary</class>
 <widget class="QWidget" name="PatientSummary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>817</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>799</width>
    <height>799</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PatientSummary</string>
  </property>
  <widget class="GraphicsView" name="teethView">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>170</y>
     <width>721</width>
     <height>451</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>721</width>
     <height>451</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="ProcedureTable" name="procedureTable">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>655</y>
     <width>721</width>
     <height>141</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>721</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="PatientTile" name="patientTile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>121</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>461</width>
     <height>121</height>
    </size>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="AllergiesTile" name="allergiesTile">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>121</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="dateSlider">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>628</y>
     <width>591</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>591</width>
     <height>20</height>
    </size>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="showLingual">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Изглед от лингвално</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showPerio">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Пародонтални измервания</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>View/ListView/TeethView/GraphicsView.h</header>
  </customwidget>
  <customwidget>
   <class>ProcedureTable</class>
   <extends>QTableView</extends>
   <header>View/ProcedureDisplayModel/ProcedureTable.h</header>
  </customwidget>
  <customwidget>
   <class>PatientTile</class>
   <extends>QPushButton</extends>
   <header>View/uiComponents/qt_derived/Widgets/TileButton.h</header>
  </customwidget>
  <customwidget>
   <class>AllergiesTile</class>
   <extends>QPushButton</extends>
   <header>View/uiComponents/qt_derived/Widgets/TileButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
